pkg/tnf/handlers/podantiaffinity/podantiaffinity_test.go

```diff
@@ -0,0 +1,141 @@
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation; either version 2 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License along
+// with this program; if not, write to the Free Software Foundation, Inc.,
+// 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+
+package podantiaffinity_test
+
+import (
+	"fmt"
+	"path"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/test-network-function/test-network-function/pkg/tnf"
+	"github.com/test-network-function/test-network-function/pkg/tnf/handlers/generic"
+	"github.com/test-network-function/test-network-function/pkg/tnf/identifier"
+	"github.com/test-network-function/test-network-function/pkg/tnf/reel"
+	"github.com/xeipuuv/gojsonschema"
+)
+
+const (
+	testTimeoutDuration = time.Second * 10
+)
+
+var (
+	genericTestSchemaFile = path.Join("schemas", "generic-test.schema.json")
+	checkSubFilename      = "podantiaffinity.json"
+	expectedFailPattern1  = "(?m)PODAAFAIL"
+	expectedPassPattern   = "(?m)OK"
+	expectedFailPattern2  = "(?m)REPLICAFAIL"
+	pathRelativeToRoot    = path.Join("..", "..", "..", "..")
+	pathToTestSchemaFile  = path.Join(pathRelativeToRoot, genericTestSchemaFile)
+	testDeploymentName    = "testDeployName123"
+	testNamespace         = "testDeployNamespace123"
+)
+
+func createTest() (*tnf.Tester, []reel.Handler, *gojsonschema.Result, error) {
+	values := make(map[string]interface{})
+	values["DEPLOYMENT_NAME"] = testDeploymentName
+	values["DEPLOYMENT_NAMESPACE"] = testNamespace
+	return generic.NewGenericFromMap(checkSubFilename, pathToTestSchemaFile, values)
+}
+
+func TestPods_Args(t *testing.T) {
+	test, handlers, jsonParseResult, err := createTest()
+
+	assert.Nil(t, err)
+	assert.True(t, jsonParseResult.Valid())
+	assert.NotNil(t, handlers)
+
+	assert.Nil(t, (*test).Args())
+}
+
+func TestPods_GetIdentifier(t *testing.T) {
+	test, handlers, jsonParseResult, err := createTest()
+
+	assert.Nil(t, err)
+	assert.True(t, jsonParseResult.Valid())
+	assert.NotNil(t, handlers)
+
+	assert.Equal(t, identifier.PodAntiAffinityIdentifier, (*test).GetIdentifier())
+}
+
+func TestPods_ReelFirst(t *testing.T) {
+	_, handlers, jsonParseResult, err := createTest()
+
+	assert.Nil(t, err)
+	assert.True(t, jsonParseResult.Valid())
+	assert.NotNil(t, handlers)
+
+	assert.Equal(t, 1, len(handlers))
+	handler := handlers[0]
+	step := handler.ReelFirst()
+	expectedReelFirstExecute := fmt.Sprintf("oc get deployment %s -n %s -o json | jq -r ' if .spec.replicas > 1 then if .spec.template.spec.affinity.podAntiAffinity != null then \"OK\" else \"PODAAFAIL\" end else \"REPLICAFAIL\" end '\n",
```

Comment:
wrap the line at a reasonable width

Reply:
You could break long string:

"line 1" +
"line 2" +
"line 3"

Reply:
changed to 3 short lines